library/src/main/java/com/google/android/exoplayer/text/webvtt/WebvttParser.java

```diff
@@ -46,6 +47,10 @@
   private static final Pattern WEBVTT_FILE_HEADER =
       Pattern.compile(WEBVTT_FILE_HEADER_STRING);
 
+  private static final String WEBVTT_COMMENT_BLOCK_STRING = "^NOTE((\\u0020|\u0009).*)?$";
```

Comment:
Heh, sorry, guess it's fine to have it how you have it already (given it's consistent with the existing WEBVTT regex). 

One question: Do you understand why The u0020 has two backslashes before it where-as the u0009 one only has one? I think I wrote that code, but it looks like a typo? Should both have only a single backslash?